### PR TITLE
setup mocha configuration

### PR DESCRIPTION
**Pull request by cgomezb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "projects",
```
Comment (aperdomob) [inline package.json]: aqui va psl-workshop-api-testing-js

File: test/HelloWord.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const { assert } = require('chai');
+
+describe('Array', () => {
+  describe('#indexOf()', () => {
+    it('should return -1 when the value is not present', function () {
+      assert.equal(-1, [1, 2, 3].indexOf(4));
+    });
+  });
+});
```
Comment (aperdomob) [inline test/HelloWord.test.js]: agregar un salto de linea

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: test/HelloWord.test.js (hunk 2 above)
Comment (cgomezb) [reply test/HelloWord.test.js]: Done

File: package.json (hunk 1 above)
Comment (cgomezb) [reply package.json]: Done

**Review by aperdomob — APPROVED** [review]

**Pull request merged**